docs/csharp/programming-guide/classes-and-structs/snippets/fields/Program.cs

```diff
@@ -3,25 +3,23 @@
 //<Snippet1>
 public class CalendarEntry
 {
-    // private field
-    private DateTime date;
 
-    // public field (Generally not recommended.)
```

Comment:
This line should stay in the source. While not recommended, it is allowed. The comment does indicate that it's not the recommended practice.

You could / should change the name to "Day" to follow the conventions.

Reply:
This public field has already been renamed to "Day" but I moved it below the property so it still exists. I moved it away from the top because I thought that it would be poor to put an example that is not recommended up top, as well as then it allows the private field to be by its wrapping property which I think could be good practice, and if that happens, the public property would have to be the first example seen if it is still up top, but I can add it back to the top again, and move the private field above the public field like originally. Does that sounds good to move it back to the original format ordering or keep it as is? I will update the .md as well.

Reply:
Thanks for clarifying @BenjaminMichaelis.  I missed it looking at the diffs. The declaration can stay where it is.